src/constants/communityCategory.ts

```diff
@@ -1,4 +1,4 @@
-import CategoryType from './category';
+import { CategoryType } from '@/types/category';
```

Comment:
"@/types"로 import 해야할거 같습니다

Reply:
```
Failed to compile.

./src/constants/communityCategory.ts + 27 modules
Cannot get final name for export 'CategoryType' of ./src/types/index.ts


> Build failed because of webpack errors
```
"@/types"로 import하게되면 위와 같은 오류가 생깁니다. IDE에서는 경로를 잘 찾아가는데 webpack에서 무언가 문제가 있습니다. 원인은 확인 해보아야 할 것 같습니다..

Reply:
넵 알겠습니다!